dss_datamover/README.md

```diff
@@ -39,7 +39,7 @@ is part of sudoers file.
 
 ```bash
 sh -c ' source  /usr/local/bin/setenv-for-gcc510.sh && python3 master_application.py PUT  '
-sh -c ' source  /usr/local/bin/setenv-for-gcc510.sh && python3 master_application.py PUT  --compaction'
+sh -c ' source  /usr/local/bin/setenv-for-gcc510.sh && python3 master_application.py PUT  --compaction YES'
```

Comment:
By default, compaction should be running. If the user says "no" explicitly, then it is disabled
Also CAPS are not ok.

Reply:
the compaction will run according to what is specified in the config file (typically yes) if no --compaction option is passed in.  See my comment below about default compaction value